google/cloud/spanner/testing/cleanup_stale_instances.cc

```diff
@@ -88,6 +101,13 @@ Status CleanupStaleInstances(Project const& project) {
 }
 
 Status CleanupStaleInstanceConfigs(Project const& project) {
+  spanner_admin::InstanceAdminClient instance_admin_client(
+      spanner_admin::MakeInstanceAdminConnection());
+  return CleanupStaleInstanceConfigs(project, instance_admin_client);
+}
```

Comment:
I'd vote for not having these compatibility wrappers, and instead update the old calls to match the new style.

Reply:
Done.